### PR TITLE
Remove fuzzy search

### PR DESCRIPTION
**Pull request by KitBurgess**
Enforce strict searching by IDs, removing the problem where a user would search for "India", and receive "Indiana", "British Indian Territories", "India"... etc.

We'll reintroduce fuzzy searching in a different PR under a different method (`fuzzy_search`) that can be used for exploratory analysis.

### CODE REVIEW

File: vortexasdk/api/shared_types.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ class IDNameLayer:
     """Triple holding `id`, `name`, and `layer`."""
 
     id: ID
-    layer: str
+    layer: List[str]
```
Comment (KitBurgess) [inline vortexasdk/api/shared_types.py]: This is not related to PR, it's a bug fix

**Comment by kujon** [comment]
Do we need to support `Union[ID, List[ID]]` for our filters? There isn't that much overhead in wrapping things in `[]`, but the code and the types are more complex as a result.

**Comment by KitBurgess** [comment]
> Do we need to support `Union[ID, List[ID]]` for our filters? There isn't that much overhead in wrapping things in `[]` when querying, but the code and the types are more complex as a result.

If we were to only support `List[ID]`, we'd still have to check that the argument was indeed a `List[ID]`, instead of a plain old `ID`. The additional complexity of doing so is no less greater than wrapping an ID in a list.
